praas-app-ionic/src/components/Form/Checkbox/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { IonCheckbox, IonLabel } from '@ionic/react';
+import { Controller, useFormContext } from 'react-hook-form';
+
+type CheckboxProps = {
```

Comment:
Can we make it the html props for checkbox? React.HTMLProps<HTMLInputElement>?

Reply:
I tried with this, But since this is a custom component, and `name` and `value` are `string | undefined` for `HTMLProps` the `react-hook-form`'s `Controller` component would start complaining, since the name and value prop it expects is `string`

Reply:
Did you also see to what react-hooks-form exposes for typing these?

Reply:
Being a custom component, I'd prefer to keep the Prop types as custom types, unless it is necessery to merge them.